CONTRIBUTING.md

```diff
@@ -3,31 +3,51 @@
 🎉 First off, thanks for taking the time to contribute! This project would not be possible without people like you. 🎉
 
 ## Code of Conduct
-By participating in this project, you're agreeing to uphold the [Mozilla Community Participation Guidelines](https://www.mozilla.org/en-US/about/governance/policies/participation/).
+By participating in this project, you're agreeing to uphold the [Mozilla Community Participation Guidelines](https://www.mozilla.org/en-US/about/governance/policies/participation/). If you need to report a problem, please contact [mikey@mozilla.com](mailto:mikey@mozilla.com).
 
 ## How Can I Contribute?
 
-### Non-code
+### Add More Sentences
 Help us add more sentences for other volunteers to read. See [issue 341](https://github.com/mozilla/voice-web/issues/341) for details.
 
-### Bug fixes and feature enhancements
+### Bug Fixes and Feature Enhancements
 All of our current issues can be found here on GitHub. Anything with a [help wanted](https://github.com/mozilla/voice-web/issues?q=is%3Aissue+is%3Aopen+label%3A%22help+wanted%22) tag is up for grabs.
 
 #### Requirements
 - [NodeJS](https://nodejs.org/en/) (v7 or higher)
 - [npm](https://www.npmjs.com/) (4 or higher)
+- [ffmpeg](https://www.ffmpeg.org/download.html )
```

Comment:
minor: there's a trailing space in the URL here. it works for me in Safari, but don't know if all browsers handle this will ;)

Reply:
Fixed!